code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -119,11 +124,12 @@
 					B.data["gender"] = T.gender
 					B.data["real_name"] = T.real_name
 					B.data["factions"] = T.faction
-					reagents.reagent_list += B
+					reagents.reagent_list = list(B) // syringes can only hold one blood reagent

```

Comment:
Syringes can hold more than one thing at a time.

Using = instead of += means you are basically removing any existing reagents in the syringe to insert blood.

While this is potentially not a problem (because it would usually just fill the syringe), you haven't changed earlier code which calculates how much blood to take.

This could cause future issues.